### PR TITLE
Serve post files from web accessible location

### PR DESCRIPTION
**Pull request by alexmorrisnz**
For #352

### CODE REVIEW

**Comment by danmarsden** [comment]
I haven't looked but is there a way to override the template without needing to change the upstream simplesaml lib? - it's easier if we can maintain a clean copy of upstreams simplesaml lib.

I'd also be tempted to check to see if the newer version of simplesamlphp (which we need to upgrade at some point anyway) - makes this any easier?

**Comment by alexmorrisnz** [comment]
You could move the post.js/css files into `sp/`, then remove the changes made to the template reverting it back to upstream.
```
<script src="/<?php echo $this->data['baseurlpath']; ?>resources/post.js"></script>
    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>resources/post.css" />
```
The files would then be served out of `/auth/saml2/sp/`

I think the newer version may make this harder as post.php is switched to the twig templating system.

File: .extlib/simplesamlphp/templates/post.php [hunk 1]
```diff
@@ -4,8 +4,8 @@
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <title>POST data</title>
-    <script src="/<?php echo $this->data['baseurlpath']; ?>../.extlib/simplesamlphp/www/resources/post.js"></script>
-    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>../.extlib/simplesamlphp/www/resources/post.css" />
+    <script src="/<?php echo $this->data['baseurlpath']; ?>../post.js"></script>
+    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>../post.css" />
```
Comment (brendanheywood) [inline .extlib/simplesamlphp/templates/post.php:8]: The original code is:
```
    <script src="/<?php echo $this->data['baseurlpath']; ?>resources/post.js"></script>
    <link type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>resources/post.css" />
```

so can we just revert back to that and put the files into here and then it should all be zero touch:

```
/auth/saml2/sp/resources/post.css
/auth/saml2/sp/resources/post.js
```

Comment (brendanheywood) [reply .extlib/simplesamlphp/templates/post.php:8]: and this should still work when we upgrade to the next version which dumps the php templates and moves to twig instead:

https://github.com/simplesamlphp/simplesamlphp/blob/master/templates/post.twig

Comment (brendanheywood) [reply .extlib/simplesamlphp/templates/post.php:8]: ah snap, didn't see the other comments, we're on the same page :)

**Comment by brendanheywood** [comment]
This is good to go when green

**Comment by alexmorrisnz** [comment]
Are the CI tests running? on my fork they only take 10~ minutes

**Pull request merged**